# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu_bindings/cbindgen.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu_bindings/cbindgen.toml b/gfx/wgpu_bindings/cbindgen.toml
--- a/gfx/wgpu_bindings/cbindgen.toml
+++ b/gfx/wgpu_bindings/cbindgen.toml
@@ -42,19 +42,19 @@ exclude = [
 "CommandEncoderDescriptor_RawString" = "CommandEncoderDescriptor"
 "RenderBundleDescriptor_RawString" = "RenderBundleDescriptor"
 "DeviceDescriptor_RawString" = "DeviceDescriptor"
 "TextureDescriptor_RawString" = "TextureDescriptor"
 "SamplerDescriptor_RawString" = "SamplerDescriptor"
 
 [parse]
 parse_deps = true
-include = ["wgpu-core", "wgpu-types"]
+include = ["wgpu-core", "wgpu-types", "wgpu-hal"]
 
-extra_bindings = ["wgpu-core", "wgpu-types"]
+extra_bindings = ["wgpu-core", "wgpu-types", "wgpu-hal"]
 
 [fn]
 prefix = "WGPU_INLINE"
 postfix = "WGPU_FUNC"
 args = "Vertical"
 rename_args = "GeckoCase"
 
 [struct]